Libraries/Components/AccessibilityInfo/AccessibilityInfo.js

```diff
@@ -33,6 +33,7 @@ type AccessibilityEventDefinitions = {
   change: [boolean], // screenReaderChanged
   reduceMotionChanged: [boolean],
   screenReaderChanged: [boolean],
+  accessibilityServiceChanged: [boolean],
```

Comment:
If this event is only supported on Android, we should mark that (perhaps by adding a new type `AccessibilityEventDefinitionsAndroid` like iOS has.